### DISCUSSION TITLE
特定网络环境下，无法上传图片

### DISCUSSION THREAD

**Discussion by Decrabbityyy** [LagrangeDev/Lagrange.Core#92] [Q&A] [answered]
docker内运行lagrange，图片无法上传

**Comment by Decrabbityyy** [comment]
给docker启动命令加一条 --dns 119.29.29.29

**Reply by wastee** [reply]
国内腾讯云的朋友可以换成 114 的，直接用 dnspod 的 dns 也 ping 不通

**Comment by Decrabbityyy** [comment]
腾讯云服务器无法发送图片，原因未知

**Reply by Shiro-ko** [reply]
请在容器中ping htdata3.qq.com，若无数据返回，也需要在启动时附加dns设置

**Comment by ghost** [comment, marked as answer]
此问题现在已经修复 无需单独配置DNS
